src/app/query/stacks/nonce/account-nonces.hooks.ts

```diff
@@ -16,6 +16,7 @@ export function useNextNonce(address: string) {
 
   return useQuery({
     ...createGetAccountNoncesQueryOptions({ address, client, network: network.chain.stacks.url }),
+    staleTime: 0,
```

Comment:
This is the fix to stop caching but maybe in cases like this we should consider just bypassing React Query and making a standard call? 

We also have an almost identical hook for this in mobile. The only difference seems to be how we get `currentNetwork`. I can add the stale time of `0` there also but maybe this hook should just be in the stacks package?

Reply:
I debated this fix and did start with an imperative call, before realising that the issue more directly lay elsewhere. IMO this is kinda of an area where RQ fails, there is no way to force show the new value. It will always render with cache first. 

Stacks package should remain react-agnostic imo, I would put the nonce calculator logic there, and only export `createGetAccountNoncesQueryOptions` from there